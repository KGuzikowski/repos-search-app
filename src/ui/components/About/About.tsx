import styled from 'styled-components'

const About = styled.div`
    font-size: 20px;
    
    @media only screen and (max-width: 700px) {
        font-size: 16px;
    }
`

export default About