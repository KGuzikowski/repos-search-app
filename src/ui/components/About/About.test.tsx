import React from 'react'
import { create } from 'react-test-renderer'
import About from './About'

it('renders About component correctly', () => {
    const about = create(<About>Testing</About>)
    expect(about.toJSON()).toMatchSnapshot()
})