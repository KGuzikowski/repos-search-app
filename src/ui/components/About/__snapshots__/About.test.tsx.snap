// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders About component correctly 1`] = `
<div
  className="sc-AxjAm clzsbK"
>
  Testing
</div>
`;
